src/core/public/chrome/utils.ts

```diff
@@ -221,3 +221,11 @@ export const getVisibleUseCases = (navGroupMap: Record<string, NavGroupItemInMap
     (navGroup) => navGroup.status !== NavGroupStatus.Hidden && navGroup.type === undefined
   );
 };
+
+export const isTruthy = (value: any) => {
+  if (Array.isArray(value) && value.length === 0) {
+    return false;
+  }
+
+  return Boolean(value);
+};
```

Comment:
The word "truthy" has an expectation in js, but the empty array check here breaks that. Can we rename this or if its only used in one file (which it looks like), can just put it there?

Reply:
will rename to hasValue, and move to the file that uses it